arkouda/io.py

```diff
@@ -1190,7 +1189,7 @@ def to_csv(
     - CSV format is not currently supported by load/load_all operations
     - The column delimiter is expected to be the same for column names and data
     - Be sure that column delimiters are not found within your data.
-    - All CSV files must delimit rows using newline (`\n`) at this time.
+    - All CSV files must delimit rows using newline (``\\n``) at this time.
     - Unlike other file formats, CSV files store Strings as their UTF-8 format instead of storing
     bytes as uint(8).
```

Comment:
we need 2 spaces here so bytes as uint(8) isn't on a new line

Reply:
updated